data/pref.xml

```diff
@@ -485,6 +485,7 @@
       <option id="filename" type="std::string" />
       <option id="resize_scale" type="double" default="1" />
       <option id="layer" type="std::string" />
+      <option id="layerIndex" type="int" default ="-1" />
```

Comment:
Please check that we are using other format in the pref.xml file:
```suggestion
      <option id="layer_index" type="int" default ="-1" />
```

Reply:
I think that this is not the first time I made this mistake...🤦
Will fix it, thanks!